input/resources/CodeSystem-ehealth-program.json

```diff
@@ -0,0 +1,37 @@
+{
+  "resourceType": "CodeSystem",
+  "id": "ehealth-program",
+  "url": "http://ehealth.sundhed.dk/cs/ehealth-program",
```

Comment:
@esn-systematic do we want "ehealth-program" or just "program". it seems like we some times append "ehealth" and sometimes not.

Reply:
This is about particular use of the infrastructure, hence ehealth-program (similar to co-existence tag ehealth-system).